modules/ipi-install-modifying-install-config-for-dual-stack-network.adoc

```diff
@@ -36,3 +39,20 @@ platform:
       - <wildcard_ipv4>
       - <wildcard_ipv6>
 ----
+
+ifdef::vSphere[]
+[IMPORTANT]
+====
+You can configure dual-stack networking on single interface only.
```

Comment:
s/single/a single

I'm not clear on what this sentence means, but I am not the target audience. I think my suggestion is right.

Reply:
yes!